demo/mmdetection_cfg/cascade_rcnn_x101_64x4d_fpn_1class.py

```diff
@@ -239,17 +239,17 @@
     workers_per_gpu=2,
     train=dict(
         type=dataset_type,
-        ann_file=data_root + 'annotations/instances_train2017.json',
-        img_prefix=data_root + 'train2017/',
+        ann_file=f'{data_root}annotations/instances_train2017.json',
```

Comment:
use ` f'{data_root}/annotations/instances_train2017.json' ` and remove the trailing ` / ` in data_root

Reply:
fixed.